component/snackbar/snackbar.css

```diff
@@ -0,0 +1,29 @@
+.snackbar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: fit-content;
+    padding: 0.5rem 2rem;
+    z-index: 5;
+}
+
+.snackbar-btn{
+    background-color: transparent;
+    color: var(--PRIMARY-COLOR);
+    cursor: pointer;
+}
+
+.snackbar-success{
+    background-color: var(--SUCCESS-BACKGROUND_COLOR);
```

Comment:
you have mixed underscore and hyphens here, you need to be bit more consistent with color names

Reply:
yeah, I noticed this while writing the code. But the name should be changed in Component Library. So, used the name as it is for now. Will change this later.